### PR TITLE
DocFX contrib

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@danroth27  @tdykstra  @spboyer 
I think we'll have far better luck if we keep this short and simple.

**Comment by Rick-Anderson** [comment]
BTW, best way to review is tap the **Files Changed** tab and tap **View**
capture

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,113 +1,29 @@
-# Contributing #
+Contributing
+============
 
-Information on contributing to this repo is in the [Contributing Guide](https://github.com/aspnet/Home/blob/dev/CONTRIBUTING.md) in the Home repo.
+The document covers the basic process for contributing to the [ASP.NET Core documentation site](https://docs.asp.net). See the [ASP.NET Core contributing guide](https://github.com/aspnet/Home/blob/dev/CONTRIBUTING.md) for information on contributing to ASP.NET Core source.
```
Comment (tdykstra) [inline CONTRIBUTING.md]: Second sentence would be clearer with "for info on..." first, followed by "see ..."

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,113 +1,29 @@
...
+## Contributing process
 
-## Video: Getting Started ##
-[Watch a video](http://ardalis.com/contributing-to-asp-net-5-documentation) showing how to get started building the documentation locally.
+Small changes to one *.MD* file can be made in the browser by tapping the **Edit on GitHub** link in the upper right corner of the browser window. For new articles and changes involving more than one file:
 
-## Building the Docs ##
+* Open an issue describing the article you wish to write or update. Get approval before you invest too much time.
+* Fork the [aspnet/docs](https://github.com/aspnet/Docs/) repo and create a branch for your article.
+* Write or update your article using [DocFX](https://dotnet.github.io/docfx/tutorial/docfx_getting_started.html#2-use-docfx-as-a-command-line-tool) in [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/). Images go in the *\<article-name>/_static* folder. Code goes in the *\<article-name>/samples* folder. It's often helpful to review an existing article to use as an template (for example, *tutorials\first-web-api.md* and the assets in *tutorials\first-web-api*).
```
Comment (tdykstra) [inline CONTRIBUTING.md]: That Docfx link isn't about writing or updating, it's about building the site. We should link to something about DFM, such as how to do snippets.
The first-web-api examples could be links.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,113 +1,29 @@
...
+* Submit a pull request (PR) from your branch to `aspnet/docs/master` and respond to PR feedback.
 
-Once you have cloned the Docs to your local machine, the following instructions will walk you through installing the tools necessary to build and test.
+## Building the docs
```
Comment (tdykstra) [inline CONTRIBUTING.md]: Add intro that explains what this means and why you need to do it.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,113 +1,29 @@
...
-    ```git clone https://github.com/aspnet/Docs.git```
- 
-4. Navigate to the Docs repo and install the build dependencies:
-
-	```cd docs```
-
-	```pip install -r requirements.txt```
-	
+```
+docfx -t default --serve
```
Comment (tdykstra) [inline CONTRIBUTING.md]: Indent code and following explanation of it so it goes with the bullet text that introduces it.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,113 +1,29 @@
...
-Below are some common pitfalls you should try to avoid:
+Notes:
 
-- Don't forget to submit an issue before starting work on an article
-- Don't forget to create a separate branch before working on your article
-- Don't update or `merge` your branch after you submit your pull request
-- Don't forget to squash your commits once your pull request is ready to be accepted
-- If updating code samples in `/samples/`, be sure any line number references in your article remain correct
+* Currently DocFX requires the .NET Framework on Windows or Mono (for Linux or macOS)
+* The local preview currently doesn't contain themes or render table correctly; the view will differ slightly from the published site
```
Comment (tdykstra) [inline CONTRIBUTING.md]: ~~Currently~~
Also doesn't do some MS Docs extensions
"correctly" doesn't fit , could omit mentioning tables as that is covered adequately by "themes"

File: CONTRIBUTING.md (hunk 2 above)
Comment (tdykstra) [inline CONTRIBUTING.md]: What about https://docs.asp.net/en/latest/contribute/style-guide.html? We should update that doc and link to it here or delete it.

Also maybe we should link to the main Docs readme at https://github.com/Microsoft/Docs/blob/master/readme.md

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,113 +1,29 @@
...
 
-Information on contributing to this repo is in the [Contributing Guide](https://github.com/aspnet/Home/blob/dev/CONTRIBUTING.md) in the Home repo.
+The document covers the basic process for contributing to the [ASP.NET Core documentation site](https://docs.asp.net). See the [ASP.NET Core contributing guide](https://github.com/aspnet/Home/blob/dev/CONTRIBUTING.md) for information on contributing to ASP.NET Core source.
 
-The documentation is built using [Sphinx](http://sphinx-doc.org) and [reStructuredText](http://sphinx-doc.org/rest.html), and then hosted by [ReadTheDocs](http://aspnet.readthedocs.org).
+## Contributing process
 
-## Video: Getting Started ##
-[Watch a video](http://ardalis.com/contributing-to-asp-net-5-documentation) showing how to get started building the documentation locally.
+Small changes to one *.MD* file can be made in the browser by tapping the **Edit on GitHub** link in the upper right corner of the browser window. For new articles and changes involving more than one file:
```
Comment (tdykstra) [inline CONTRIBUTING.md]: md should be lower case

In Docs the button just has "Edit", not "on GitHub", and in narrow windows it's in a collapsed-by-default "options" bar

File: CONTRIBUTING.md (hunk 1 above)
Comment (Rick-Anderson) [inline CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (Rick-Anderson) [inline CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md (hunk 3 above)
Comment (Rick-Anderson) [inline CONTRIBUTING.md]: I changed the heading to "Build and test your article with DocFX"
I don't want to add an intro just to add an intro.

File: CONTRIBUTING.md (hunk 4 above)
Comment (Rick-Anderson) [inline CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,113 +1,29 @@
...
 
-The documentation is built using [Sphinx](http://sphinx-doc.org) and [reStructuredText](http://sphinx-doc.org/rest.html), and then hosted by [ReadTheDocs](http://aspnet.readthedocs.org).
+## Contributing process
 
-## Video: Getting Started ##
-[Watch a video](http://ardalis.com/contributing-to-asp-net-5-documentation) showing how to get started building the documentation locally.
+Small changes to one *.md* file can be made in the browser by tapping the **Edit** link in the upper right corner of the browser window (you might need to expand the **options** in small browser windows). For new articles and changes involving more than one file:
 
-## Building the Docs ##
+* Open an issue describing the article you wish to write or update. Get approval before you invest too much time.
```
Comment (tdykstra) [inline CONTRIBUTING.md]: Could make "Open an issue" a link to https://github.com/aspnet/Docs/issues

**Comment by spboyer** [comment]
We should have instructions or links to setting this up on OSX and Linux

**Pull request closed without merging**